rest-server/src/main/resources/static/index.html

```diff
@@ -167,12 +167,16 @@
                 appServiceRuntimes: [
                     {id: 'DOCKER', name: 'Docker with a Dockerfile'},
                     {id: 'DOCKER_SPRING', name: 'Docker with Spring Boot'},
+                    {id: 'DOCKER_MICRONAUT_GRADLE', name: 'Docker with Micronaut and Gradle'},
+                    {id: 'DOCKER_MICRONAUT', name: 'Docker with Micronaut and Maven'},
                     {id: 'JAVA', name: 'Java with Maven'},
                     {id: 'JAVA_GRADLE', name: 'Java with Gradle'},
                     {id: 'SPRING', name: 'Spring Boot (Java) with Maven'},
                     {id: 'SPRING_GRADLE', name: 'Spring Boot (Java) with Gradle'},
                     {id: 'QUARKUS', name: 'Quarkus (Java) with Maven'},
                     {id: 'QUARKUS_NATIVE', name: 'Quarkus (Native) with Maven'},
+                    {id: 'MICRONAUT_GRADLE', name: 'Micronaut with Gradle'},
+                    {id: 'MICRONAUT', name: 'Micronaut with Maven'},
```

Comment:
Due to Brand guidelines, replace :

`Micronaut with Maven`

with: 

`Micronaut Framework with Maven`